functions/src/handlers/GetProfileHandler.ts

```diff
@@ -65,7 +65,27 @@ export function getprofile (req, res) {
                     universities[uni].forEach((cuid) => {
                         if (eventsData[cuid]) {
                             userData.universities[uni][cuid].events = {}
```

Comment:
This doesnt look like it gets used since you overwrite it.

Reply:
gave error without it